src/App.js

```diff
@@ -1,44 +1,94 @@
 import React, { Component } from 'react';
 import { View, StyleSheet, Platform } from 'react-native';
-import { NewsDetails, NewsList } from './screens';
+import { ScreenNewsDetails, ScreenNewsList, ScreenWebArticle } from './screens';
 
 export default class App extends Component {
   constructor(props) {
     super(props);
 
     this.state = {
-      isNewsSelected: false,
+      showScreenNewsList: true,
+      showScreenNewsDetails: false,
+      showScreenWebArticle: false,
       article: {},
+      articles: [],
     };
     this.handleNewsSelected = this.handleNewsSelected.bind(this);
     this.handleDetailsClose = this.handleDetailsClose.bind(this);
+    this.handleOpenWebArticle = this.handleOpenWebArticle.bind(this);
+    this.handleWebArticleClose = this.handleWebArticleClose.bind(this);
   }
 
-  handleNewsSelected(article) {
-    this.setState({ article, isNewsSelected: true });
+  handleNewsSelected(article, articles) {
```

Comment:
Why do you need `articles` here?

Reply:
Explained, will be removed in the next pull request.